### PR TITLE
commit soumaya

### PR DESCRIPTION
**Pull request by soumaya-belghali**

### CODE REVIEW

File: R/bus_buff.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#' @description 
+#' This function reads the "clean_bus" `shp` file named 
+#' `bus_buff` stored in `data/`.
+#'
+#'
+#' @return bus_buff `shp`  
+#' @export
+
+
+bus_buff<- function (clean_bus) {
```
Comment (killiangregory) [inline R/bus_buff.R:13]: Amazing!

File: R/clean_bus_data.R [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#' @description 
+#' This function reads the bus `shp` file named 
+#' `bus.shp` stored in `data/`.
+#'
+#'
+#' @return A `shp`
+#' 
+#' @export
+
+bus_clean<- function (bus){
```
Comment (killiangregory) [inline R/clean_bus_data.R:12]: Wonderful!

**Review by killiangregory — APPROVED** [review]
Great!

**Review by nataliafmedeiros — APPROVED** [review]
Super!

**Pull request merged**